dynamic-loader/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../Makefile.include
+
+SOURCES         = $(shell find *.c ! -name "sm1.c" -printf "%p ")
+OBJECTS         = $(SOURCES:.c=.o)
+SM1_HEADER      = $(shell python3 hardcode_elf.py)
```

Comment:
better to use a proper Make target for this as a dependency

current approach also fails in CI:

https://github.com/sancus-tee/sancus-examples/pull/16/checks?check_run_id=3747451902